Source/Core/AudioCommon/OpenALStream.cpp

```diff
@@ -247,8 +250,8 @@ void OpenALStream::SoundLoop()
     // Remove the Buffer from the Queue.
     if (numBuffersProcessed)
     {
-      ALuint unqueuedBufferIds[OAL_MAX_BUFFERS];
-      alSourceUnqueueBuffers(uiSource, numBuffersProcessed, unqueuedBufferIds);
+      std::vector<ALuint> unqueuedBufferIds(OAL_MAX_BUFFERS);
```

Comment:
This should be `std::array` in this commit. In the next commit, you can avoid the overhead of `std::vector` by using a `std::unique_ptr<ALuint[]>`.